lib/potassium/recipes/node.rb

```diff
@@ -0,0 +1,27 @@
+require 'net/http'
+require 'semantic'
+require 'pry'
+require 'json'
+
+class Recipes::Node < Rails::AppBuilder
+  def create
+    info "Using node version #{version_alias}"
+    create_file '.node-version', version_alias, force: true
+    byebug
+    json_file = File.read(Pathname.new("package.json"))
+    js_package = JSON.parse(json_file)
+    js_package["engines"] = {"node"=> "#{version}"}
```

Comment:
Space inside { missing.<br>Surrounding space missing for operator =>.<br>Prefer to_s over string interpolation.<br>Space inside } missing.